### PR TITLE
Tweaks to registration.

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2022-01-26 at 9 54 03 AM
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 1]
```diff
@@ -444,67 +458,94 @@ const Register = ({ classes, displayErrorMessage }: IRegister) => {
...
-            You can use your credentials from SUBNET to register.{" "}
-            <Link
-              className={classes.link}
-              color="inherit"
-              target="_blank"
-              href="https://min.io/product/subnet"
-            >
-              Learn more about SUBNET →.
-            </Link>
+            Use your MinIO Subscription Nework login credentails to register
```
Comment (jinapurapu) [inline portal-ui/src/screens/Console/Support/Register.tsx:461]: "credentials"

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Support/Register.tsx]: great catch, also `Nework` hahaha

**Review by Alevsk — APPROVED** [review]

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 2]
```diff
@@ -444,67 +458,94 @@ const Register = ({ classes, displayErrorMessage }: IRegister) => {
...
+                </Button>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item xs={12} className={classes.subnetDescription}>
+            <div className={classes.smallBoxy}>
+              <h1 className={classes.headerStyle}>Why should I register?</h1>
+              Registering this cluster with the MinIO Subscription Network
+              (SUBNET) provides the following benefits in addition to the
+              commercial license and SLA backed support.
```
Comment (jinapurapu) [inline portal-ui/src/screens/Console/Support/Register.tsx:541]: Should this end in ":" instead of "." ?

**Review by Alevsk — APPROVED** [review]

File: portal-ui/src/screens/Console/Support/Register.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Support/Register.tsx]: yes!

Comment (bexsoft) [inline portal-ui/src/screens/Console/Support/Register.tsx]:
```suggestion
              commercial license and SLA backed support:
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Support/Register.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Support/Register.tsx]: done

**Review by bexsoft — DISMISSED** [review]
LGTM

**Comment by cniackz** [comment]
Hello Daniel, I am testing the branch in my computer.
The page is loading but I see console error that maybe is not related.
But just wonder if we can repair here or in another PR...

Screen Shot 2022-01-26 at 1 13 57 PM

**Review by Alevsk — APPROVED** [review]

**Review by jinapurapu — APPROVED** [review]

**Comment by cniackz** [comment]
Nice, re-tested and console error is gone:

Screen Shot 2022-01-26 at 1 29 41 PM

Thanks Daniel!

**Pull request merged**